src/models/ratings/dto/ratings.dto.ts

```diff
@@ -0,0 +1,68 @@
+import { ApiProperty } from "@nestjs/swagger";
+export class Rating {
+	@ApiProperty()
+		id: number;
+	@ApiProperty()
+		name: string;
+	@ApiProperty()
+		rating: number;
+	@ApiProperty()
+		maxrating: number;
+	@ApiProperty()
+		ratedgames: number;
+	@ApiProperty()
+		isbot: boolean;
+}
+
+export class RatingList {
+	@ApiProperty()
+		items: Array<Rating>;
+	@ApiProperty()
+		total: number;
+	@ApiProperty()
+		perPage: number;
+	@ApiProperty()
+		page: number;
+	@ApiProperty()
+		totalPages: number;
+}
+
+export class RatingQuery {
+	@ApiProperty({
+		description: 'Number of results per page',
+		required: false
+	})
+		limit?: string;
+	@ApiProperty({
+		description: 'page number',
+		required: false
+	})
+		page?: string;
+	@ApiProperty({
+		description: 'Optional skip if you dont use page',
```

Comment:
what is `skip` for? What values can I pass in?

Reply:
it's the total number of items to skip when doing pagination, can clean up this description